components/odh-notebook-controller/config/manager/manager.yaml

```diff
@@ -25,7 +25,7 @@ spec:
           imagePullPolicy: Always
           command:
             - /manager
-          args: ["--oauth-proxy-image", "registry.redhat.io/openshift4/ose-oauth-proxy@sha256:4f8d66597feeb32bb18699326029f9a71a5aca4a57679d636b876377c2e95695"]
+          args: ["--oauth-proxy-image", "registry.redhat.io/openshift4/ose-oauth-proxy-rhel9@sha256:d3056b35d9a205b9f2c48d924f199c5ac23904eb18d526e4bff229e7c7181415"]
```

Comment:
this seems to be very different sha compared to main/2.24 branch
https://github.com/red-hat-data-services/kubeflow/commit/dc064a08bea1b9d45748d82de8c88c6030dca9ab
additionally, this doesnt match the additional-image-patch
https://github.com/red-hat-data-services/RHOAI-Build-Config/blob/87b0d3df8bca0804095de7e481d22d47f06f2828/bundle/additional-images-patch.yaml#L20

is this intentional ?

Reply:
Yep, as you can see in my commits here - there are two that are simply cherry-picks and the third one that updates to the latest image. I wrote this info also on the jira in the comment so we can agree on what to merge here - whether to use outdated image that is matching what goes out in 2.24 or whether to use the latest available at this point of time.

Note: the additional patch - I'm not sure what branch will be used for the release as there is this branch https://github.com/red-hat-data-services/RHOAI-Build-Config/blob/2.22.1-catalog-patch/bundle/additional-images-patch.yaml. We should probably ask too, so we are sure there.

Reply:
So, we got merged this change rhoai-additional-images#60, so I presume we should feel safe of having this new image in place for 2.22, WDYT?